### PR TITLE
Swift4 xcode11

### PR DESCRIPTION
**Pull request by matelo**

### CODE REVIEW

File: Sources/ContentfulSyncSerializer/SyncJSONDownloader.swift [hunk 1]
```diff
@@ -6,97 +6,137 @@
...
+                        case let .error(error):
+                            completion(.error(error))
+                        }
                     }
+                case let .error(error):
+                    completion(.error(error))
+                }
+            }
+        }
+    }
```
Comment (jroehl) [inline Sources/ContentfulSyncSerializer/SyncJSONDownloader.swift]: Is there anything we can do about the callback hell? 😅

**Review by jroehl — APPROVED** [review]
one minor comment - otherwise it looks good to me 👍

File: Sources/ContentfulSyncSerializer/SyncJSONDownloader.swift (hunk 1 above)
Comment (matelo) [reply Sources/ContentfulSyncSerializer/SyncJSONDownloader.swift]: yeah, I can refactor it, to use some reactive way of handling

**Review by jroehl — APPROVED** [review]
Very nice - thanks for refactoring the callback madness!

**Review by hwartig — APPROVED** [review]

**Pull request merged**